script/bootstrap

```diff
@@ -0,0 +1,6 @@
+brew list nodenv >> /dev/null || brew install nodenv
+if grep -q "nodenv init -" ~/.zshrc ; then
+   echo '✅ .zshrc already inits nodenv, good.'
+else
+   echo '⚠️  Manually add the following to your .zshrc: eval "$(nodenv init -)"'
```

Comment:
My only thought here would be that not everybody is a zsh user, so this is a bit fragile. No immediate thoughts on how to make it better though.

We could move it to the Dockerfile to actually make the file change though, since this should (I think) happen as part of spinning up the container, and we then develop entirely within that?

Reply:
Thought about this and pinned Dockerfile to 14.

Reply:
This is fragile, which is one of the reasons why I didn't edit the config file directly, also it could be .bashrc, .bashprofile, .zprofile.
Changing the text to make it vaguer feels unhelpful; probably not work the time.